packages/venia-concept/src/components/SearchBar/suggestions.js

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment } from 'react';
+
+import { mergeClasses } from 'src/classify';
+import SuggestedCategories from './suggestedCategories';
+import SuggestedProducts from './suggestedProducts';
+import defaultClasses from './suggestions.css';
+
+const Suggestions = props => {
+    const { products, searchValue, visible } = props;
+    const { filters, items } = products;
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    if (!visible || !filters || !items) {
+        return null;
+    }
+
+    const categoryFilter =
+        filters.find(({ name }) => name === 'Category') || {};
+    const categories = categoryFilter.filter_items || [];
+
+    return (
+        <Fragment>
+            <SuggestedCategories categories={categories} value={searchValue} />
+            <h2 className={classes.heading}>
+                <span>{'Product Suggestions'}</span>
```

Comment:
Could use the literal here.

```suggestion
                <span>Product Suggestions</span>
```

Reply:
I'd prefer to avoid literal text nodes. It'll help with translation down the line, plus

- they just get converted to strings anyway
- they trim whitespace aggressively
- they tend to mess with syntax highlighting

Reply:
I don't _love_ this, but I see all those points, especially the i18n one.